client/src/PersonalRecipe.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import MaterialTable from 'material-table';
+import './PersonalRecipe.css'
+import { forwardRef } from 'react';
+import ArrowUpward from '@material-ui/icons/ArrowUpward';
+import Check from '@material-ui/icons/Check';
+import Clear from '@material-ui/icons/Clear';
+import DeleteOutline from '@material-ui/icons/DeleteOutline';
+import Edit from '@material-ui/icons/Edit';
+
+const tableIcons = {
+    Check: forwardRef((props, ref) => <Check {...props} ref={ref} />),
+    Clear: forwardRef((props, ref) => <Clear {...props} ref={ref} />),
+    Delete: forwardRef((props, ref) => <DeleteOutline {...props} ref={ref} />),
+    SortArrow: forwardRef((props, ref) => <ArrowUpward {...props} ref={ref} />),
+    Edit: forwardRef((props, ref) => <Edit {...props} ref={ref} />),
+  };
+
+export default function PersonalRecipe() {
+  const [state, setState] = React.useState({
+    columns: [
+      { title: 'Food', field: 'name' },
+      // { title: 'Price', field: 'birthYear', type: 'numeric' },
+    ],
+    data: [
+      { name: 'Banh mi', birthYear: 1987 },
+      {
+        name: 'Pho',
+        birthYear: 2017
+      },
+    ],
```

Comment:
I'm assuming you will work on this more in a future PR? If so, maybe we can add images to the side?

Reply:
yes